extension/src/plots/model/index.ts

```diff
@@ -105,6 +106,10 @@ export class PlotsModel extends ModelWithPersistence {
   }
 
   public async transformAndSetPlots(data: PlotsOutput, revs: string[]) {
+    if (data.error) {
```

Comment:
[F] Related to this comment: https://github.com/iterative/vscode-dvc/issues/1649#issuecomment-1295969178. Will fix separately.